data/ui/ButtonDialog.ui

```diff
@@ -250,4 +251,31 @@
       <action-widget response="apply" default="true">apply</action-widget>
     </action-widgets>
   </template>
+  <object class="GtkListBoxRow" id="row_left_hand_mode">
+    <property name="width_request">100</property>
+    <property name="visible">True</property>
+    <property name="can_focus">True</property>
+    <child>
+      <object class="GtkBox">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="margin_top">4</property>
+        <property name="margin_bottom">4</property>
+        <child>
+          <object class="GtkLabel">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="halign">start</property>
+            <property name="label" translatable="yes">Switch the left- and right mouse buttons</property>
```

Comment:
I think I want two dialogs, the current one for all buttons but left/right and one that essentially just has a on/off with an explanation below/next to it/wherever. So what you propose, swapping out the listbox with the other widget is the right thing, yes.